Sources/SwiftGenKit/Parsers/StringsParser.swift

```diff
@@ -37,23 +37,58 @@ public final class StringsParser: Parser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parse(path: Path) throws {
-    let name = path.lastComponentWithoutExtension
+    if path.isDirectory {
 
-    guard tables[name] == nil else {
-      throw StringsParserError.duplicateTable(name: name)
-    }
-    guard let data = try? path.read() else {
-      throw StringsParserError.failureOnLoading(path: path.string)
-    }
+      if try hasMultipleLprojSubdirsWithDifferentLanguages(path: path) {
+        warningHandler?("Multiple .lproj directories found with different languages." +
+          "Duplicate keys will be parsed multiple times.", #file, #line)
```

Comment:
`#file` and `#line` are not necessary here, they are already passed by the magic of default values for those parameters being resolved at call site 😉